packages/bygger/server/src/GitHubRepo.js

```diff
@@ -26,12 +32,12 @@ export class GitHubRepo {
       });
       this.authentication = await auth({
         type: "installation",
-        installationId: process.env.GITHUB_APP_INSTALLATION_ID,
+        installationId: this.credentials.installationId,
       });
       logger.debug("Authenticate on Github as app installation");
     }
     this.octokit = new Octokit({
-      auth: this.authentication?.token ?? "",
+      auth: this.authentication?.token ?? this.credentials.token,
```

Comment:
Logg med error hvis ikke this.credentials.token eller this.authentication?.token er satt. Hovedsaklig for localhost, men fint å ta det som error uansett